### DISCUSSION TITLE
Xcode 14.3 and macOS 13 Ventura already here!

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#1518] [Announcements]
**Xcode 14.3** is now available on our macOS M1 build machines. This version will be used to run the build if you specify `14.3` as the Xcode version.

You can configure the Xcode version in the Build section of Flutter workflow editor or in the `environment` section of codemagic.yaml.
See the [build machine specs](https://docs.codemagic.io/specs/versions-macos/) for information about the preinstalled software and available runtimes.

**Comment by helinanever** [comment]
Update: `latest` and `edge` in Xcode version settings now point to 14.3.
